### PR TITLE
Store prefix and block in file table and FileMetadata

### PR DESCRIPTION
**Pull request by L-Henke**
Adapt documentation to changes in https://github.com/Qabel/qabel-android/pull/248.

@jmt85 @julianseeger 
We should discuss why we made this change and if we really want it.

### CODE REVIEW

File: _docs/Qabel-Protocol-Box.md [hunk 1]
```diff
@@ -646,6 +649,7 @@ CREATE TABLE shares
 /*
 Table of all file objects in the directory
 * 'id' is meaningless and only for record keeping purposes.
+* 'prefix is the the prefix to the block
```
Comment (roeslpa) [inline _docs/Qabel-Protocol-Box.md]: Typo: the the

File: _docs/Qabel-Protocol-Box.md [hunk 2]
```diff
@@ -717,6 +722,7 @@ CREATE TABLE spec_version
 Table for the file information
 * 'id' is meaningless and only for record keeping purposes.
 * 'owner' is the public key of the owner
+* 'prefix is the the prefix to the block
```
Comment (roeslpa) [inline _docs/Qabel-Protocol-Box.md]: Typo: the the

**Comment by jan-schreib** [comment]
Whats the status on this? @L-Henke please poke the right people if you need input here.

**Comment by jan-schreib** [comment]
@L-Henke if you are not responsible any more, please assign someone and provide a status update.

**Pull request closed without merging**